### ISSUE TITLE
Window goes 1 pixel over screen on multi-monitor setups

### ISSUE THREAD

**Issue by nobbele** [Quaver/Quaver#863]
**Describe the bug**
Quaver window goes 1 pixel over to the second monitor

**To Reproduce**
1. Get a second monitor
2. Open quaver 

**Expected behavior**
Quaver is displayed on monitor 1 only, no extra pixels on monitor 2

**Desktop:**
 - OS: Windows 10

**Comment by YaLTeR** [comment]
Is it fullscreen? If not then that might be just the window border.

**Comment by nobbele** [comment]
It's not fullscreen

**Comment by YaLTeR** [comment]
Yeah, well, might just be the window border then? Can't do much about that.

**Comment by nobbele** [comment]
Other programs don't have this problem though?

**Comment by YaLTeR** [comment]
If you resize another windowed program so that the program contents are exactly your monitor size it will also have the window border sticking out.

**Comment by nobbele** [comment]
You can do something about the window border, `Game.Window.IsBorderless = true;` fixes this problem.
